csfieldguide/chapters/content/en/network-communication-protocols/sections/transport-layer-protocols.md

```diff
@@ -31,12 +31,12 @@ So each page is split up and sent in tiny pieces, then at the other end they are
 
 {panel type="curiosity"}
 
-# Three days?!
+# A full day?!
 
-At 86 terabytes of code, Google's database is one of the largest in the world.
-To write that all down and store in 40ft shipping containers, would create a train as long as the distance from Barcelona to Moscow!
+According to one [2016 estimate](https://www.gearthblog.com/blog/archives/2016/04/big-google-earth-database.html), Google Earth has approximately 3 petabytes (3000000Gb) of past and present image data on our world from above.
+To write that all down and store in 40ft shipping containers would create a train over a thousand kilometers long &ndash; as long as the distance from Auckland to Dunedin!
```

Comment:
Should we stick to one measurement system? Maybe change 40ft to 12m?

Reply:
I'm using '40ft shipping container' to refer to the standard rather than the length. I think most metric system users would understand
https://en.wikipedia.org/wiki/Intermodal_container
I can change it if you want, but I can't say 12m (I'd need to use 12.19) because that's significant enough to make me redo all the calculations

Reply:
What does the calculation come to if you use 12m? You've used assumptions elsewhere in the calculation so I'm sure rounding it is fine as you're not giving an exact result anyway. I think it's neater to keep the same units especially as you should work in the same units when doing caluclations anyway.

Reply:
Okay I'll recalculate it and get back to you

Reply:
I recalculated both and the difference isn't too significant, however I don't like using containers that don't quite fit the standard so I'm keeping the actual measurements from the 40ft version. As for what to write, I reworded it a little so the `12m` is in there, but if you insist I'll remove the 40ft part

Reply:
I'm fine with both in there